ledger-service/http-json/src/test/scala/com/digitalasset/http/util/InsertDeleteStepTest.scala

```diff
@@ -59,7 +59,7 @@ object InsertDeleteStepTest {
   implicit val `Alpha arb`: Arbitrary[Cid] = Cid subst Arbitrary(
     Gen.alphaUpperChar map (_.toString))
 
-  private implicit val `IDS monoid`
+  private[util] implicit val `IDS monoid`
```

Comment:
should we move into `InsertDeleteStep` companion object? It is a valid Monoid definition, we don't use it in prod code, but I think it is a good property to have, plus we rely on this property in testing.

Reply:
Maybe; I'll think about whether the instance is parametric in `D`, and move it (and the other) if so.